src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChat.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import ExpensesChart from './ExpensesChart';
+import type { ValuesDataWithBorder } from '@ses/core/models/dto/chart.dto';
+import type { ComponentMeta, Story } from '@storybook/react';
+import type { FigmaParams } from 'storybook-addon-figma-comparator/dist/ts/types';
+
+export default {
+  title: 'Components/Finances/ExpensesChart',
+  component: ExpensesChart,
+  decorators: [
+    (Story: Story) => (
+      <div>
+        <Story />
+      </div>
+    ),
+  ],
+  parameters: {
+    chromatic: {
+      viewports: [375, 834],
+      pauseAnimationAtEnd: true,
+    },
+  },
+  argTypes: {},
+} as ComponentMeta<typeof ExpensesChart>;
+
+const args = [
+  {
+    monthly: [
+      {
+        period: '1',
+        value: 17912500,
+      },
+      {
+        period: '1',
+        value: 17912500,
+      },
+      {
+        period: '1',
+        value: 172312500,
+      },
+      {
+        period: '1',
+        value: 17922500,
+      },
+    ],
+    newActual: [
+      {
+        itemStyle: {
+          borderRadius: [0, 0, 6, 6],
+        },
+
+        name: 'Active Budget',
+        value: 16965122,
+      },
+    ] as ValuesDataWithBorder[],
+
+    newDiscontinued: [],
+    newPrediction: [],
```

Comment:
You should add more data to see how the component will looks like in a real environment